conformance/lang/expressions/literals/boolean_literal.balt

```diff
@@ -0,0 +1,274 @@
+Test-Case: output
+Description: Test boolean literals with boolean type descriptor.
+Labels: boolean
+
+public function main() {
+    boolean x = true;
+    io:println(x); // @output true
+
+    x = false;
+    io:println(x); // @output false
+}
+
+Test-Case: output
+Description: Test boolean literals with toString().
+Labels: boolean-literal, value:toString
+
+public function main() {
+    io:println(true.toString()); // @output true
+
+    io:println(false.toString()); // @output false
+}
+
+Test-Case: output
+Description: Test boolean literals with toBalString().
+Labels: boolean-literal, value:toBalString
+
+public function main() {
+    io:println(true.toBalString()); // @output true
+
+    io:println(false.toBalString()); // @output false
+}
+
+Test-Case: output
+Description: Test boolean literals with function calls.
+Labels: boolean, function-call-expr
```

Comment:
```suggestion
Labels: boolean
```

This is under minimal subset.

Reply:
Since we are specifically testing these here shall we include the labels even though they are part of the minimal subset?

Reply:
Since we are specifically testing function call in this test, @MaryamZi suggested to add this label for these types of tests.